style.css

```diff
@@ -0,0 +1,82 @@
+body {
+background-color: white;
+font-family: arial;
+}
+
+.img {
```

Comment:
Okay,  I've make the correction using {width: 100%, text-align: center;} on class .img 

I've reloaded the files for your review. Cheers, Jennifer